code/go/0chain.net/chaincore/transaction/entity.go

```diff
@@ -412,10 +412,7 @@ func (t *Transaction) GetSummary() *TransactionSummary {
 - applicable only when running in test mode and the transaction_data string contains debug keyword somewhere in it
 */
 func (t *Transaction) DebugTxn() bool {
-	if !config.Development() {
-		return false
-	}
-	return strings.Contains(t.TransactionData, "debug")
+	return true
```

Comment:
We don't want to enable the txn debug logs by default, it will make the log files rotated too fast and lost the useful logs when error happens at the beginning